app/Jobs/SendDailyDigestNotificationsJob.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace App\Jobs;
+
+use App\Mail\TaskDigestMail;
+use App\Models\V2\Tasks\Task;
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Foundation\Bus\Dispatchable;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Queue\SerializesModels;
+use Illuminate\Support\Facades\Mail;
+
+class SendDailyDigestNotificationsJob implements ShouldQueue
+{
+    use Dispatchable;
+    use InteractsWithQueue;
+    use Queueable;
+    use SerializesModels;
+
+    private $task;
+
+    /**
+     * Create a new job instance.
+     */
+    public function __construct(Task $task)
+    {
+        $this->task = $task;
+    }
+
+    /**
+     * Execute the job.
+     */
+    public function handle(): void
+    {
+        $users = $this->task->project->users()->get();
+        foreach ($users as $user) {
+            Mail::to($user->email_address)->send(
+                new TaskDigestMail($user, $this->task)
```

Comment:
This is going to result in a huge volume of email. I would be shocked if we don't get API errors from Amazon and likely need to space these sends out in delayed chunks, which is a pretty big change for this PR. Just dropped a question in Slack to see if we can delay this feature till the next release.